resources/js/common/components/SubscribeToggleButton/SubscribeToggleButton.tsx

```diff
@@ -49,8 +54,10 @@ export const SubscribeToggleButton: FC<SubscribeToggleButtonProps> = ({
   };
 
   return (
-    <BaseButton size="sm" onClick={handleClick}>
-      {isSubscribed ? t('Unsubscribe') : t('Subscribe')}
+    <BaseButton size="sm" onClick={handleClick} className="gap-1.5" aria-label={label}>
+      {isSubscribed ? <LuBellOff className="size-4" /> : <LuBell className="size-4" />}
+
+      <span className="hidden sm:block">{label}</span>
```

Comment:
The dropdown is still truncated if there's a sufficient number of pages.
![image](https://github.com/user-attachments/assets/77d399d5-afa5-4c20-a19b-3d9d81db667c)

Reply:
Good catch. Should be fixed in latest.